sycl/unittests/assert/assert.cpp

```diff
@@ -145,10 +145,12 @@ static AssertHappened ExpectedToOutput = {
 };
 
 static constexpr int KernelLaunchCounterBase = 0;
-static int KernelLaunchCounter = KernelLaunchCounterBase;
 static constexpr int MemoryMapCounterBase = 1000;
 static int MemoryMapCounter = MemoryMapCounterBase;
+#ifndef _WIN32
```

Comment:
For other reviewers - looks like we have some tests excluded on `_WIN32` making these static functions unused. I suppose icx emits a warning about that and then it gets turned into a error due to `-Werror`.

Reply:
yeah tbh i dont know why these dont error with `cl`, but they seem unused always on windows so this change should be safe